### DISCUSSION TITLE
How to view the actual path after forwarding in access.log?

### DISCUSSION THREAD

**Discussion by 1798972** [Kong/kong#9539] [Help]
like this:
Route:
![image](https://user-images.githubusercontent.com/49218070/195242449-a9ed3fe3-4ab6-41a2-a401-42ba8bd9fca3.png)
Service:
![image](https://user-images.githubusercontent.com/49218070/195242560-64f3e81e-202f-483a-a07d-72ca38b0ab42.png)
http://kongIp:8000/echo -> http://127.0.0.1/kong/ext/echo2
---
How to display it in the access.log?

**Comment by StarlightIbuki** [comment]
Unfortunately, it cannot be done with configurations for now.
It should be possible to add this feature with some hack: in the `access` phase we have already got the upstream URL needed. If we put the URL into an Nginx variable it should be accessible from the `log_format` and `access_log` directives.
We are discussing it.
